mne/io/eeglab/eeglab.py

```diff
@@ -269,22 +315,42 @@ def __init__(self, input_fname, montage, preload=False, eog=(),
                 orig_format='double', verbose=verbose)
         else:
             if preload is False or isinstance(preload, string_types):
-                warn('Data will be preloaded. preload=False or a string '
-                     'preload is not supported when the data is stored in the '
-                     '.set file')
+                warnings.warn('Data will be preloaded. preload=False or a'
+                              ' string preload is not supported when the data'
+                              ' is stored in the .set file')
             # can't be done in standard way with preload=True because of
             # different reading path (.set file)
             data = eeg.data.reshape(eeg.nbchan, -1, order='F')
             data = data.astype(np.double)
             data *= CAL
+            data = np.vstack((data, self._event_ch))
```

Comment:
zeros, but really you can use `np.empty()` since you know you will fill all entries